core/src/main/scala/org/apache/spark/sql/sources/jdbcExtensions.scala

```diff
@@ -304,6 +304,24 @@ object JdbcExtendedUtils extends Logging {
     sql.toString()
   }
 
+  /**
+   * Returns the SQL for prepare to insert or put rows into a table.
+   */
+  def getDeleteString(table: String, rddSchema: StructType): String = {
+    val sql = new StringBuilder()
+    sql.append(s"DELETE FROM $table WHERE ")
+    var fieldsLeft = rddSchema.fields.length
+    rddSchema.fields.foreach { field =>
+      sql.append(field.name).append('=').append('?')
```

Comment:
quote the names for case-sensitivity/special chars changes in https://github.com/SnappyDataInc/snappydata/pull/648